src/sbt-test/sbt-autoversion/simple/test

```diff
@@ -0,0 +1,104 @@
+# Generate project files.
+> reload
+> compile
+
+# Setup repository.
+$ exec git init
+$ exec git add .
+$ exec git commit -m 'initial commit'
+$ exec git tag v0.0.0
+
+# Prove sbt can start.
+> reload
+> compile
+> show version
```

Comment:
Is that needed, given that the first lines of your test run a first command and that the next commands would fail if there was an issue with sbt ?

Reply:
`compile` (or really any initial command) was needed when I was figuring out the `.gitignore` for the test scenario. `show version` was only useful for debugging the tests.

Neither are needed now, I'll delete this whole first chunk.